bleachbit/Special.py

```diff
@@ -42,10 +42,10 @@ def __shred_sqlite_char_columns(table, cols=None, where=""):
     cmd = ""
     if cols and options.get('shred'):
         cmd += "update %s set %s %s;" % \
-            (table, ",".join(["%s = randomblob(length(%s))" % (col, col)
```

Comment:
This is not related to Mac OS X, and I would like to fix it a different way, please.